### PR TITLE
Create Component for Emoji picker

### PR DESCRIPTION
**Pull request by isurewould**
# What
-adding: https://github.com/ealush/emoji-picker-react as emoji picker

# Why
We will slap an emoji on the button.

# To Do:
try and get the alignment unborked. Its pushing the button up

### CODE REVIEW

**Comment by isurewould** [comment]
@arnnlee  made the picker component a class and cleaned up state logic. Need to work on formatting but now code is cleaner and no extra stuff is needed in home component.

File: src/components/Home.js [hunk 1]
```diff
@@ -20,7 +21,7 @@ class Home extends React.Component {
                     <Container maxWidth="sm">
                     <Grid container spacing={1} alignItems="center" style={{ minHeight: '100vh' }}>
                         <Grid item xs={2}>
-                        <AccountBalanceWalletIcon />
+      <ReactPicker />
```
Comment (arnnlee) [inline src/components/Home.js:24]: 👍🏼👍🏼

**Review by arnnlee — APPROVED** [review]

**Comment by arnnlee** [comment]
Looks good 👍🏼

**Comment by isurewould** [comment]
will open another branch for formatting the picker this just places the  smiley face emoji in the component, tracks the state, and renders it on home component

**Pull request merged**